compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -209,6 +215,28 @@ void RecordMinMax::initialize(const std::string &input_model_path)
   _interpreter->attachObserver(_observer.get());
 }
 
+void RecordMinMax::initialize_with_parallel_record(const std::string &input_model_path)
+{
+  initializeCircleModule(input_model_path);
+
+  // Create and initialize interpreters and observers
+  const auto threads_size = omp_get_max_threads();
+
+  _vector_interpreters.resize(threads_size);
+  _vector_observers.resize(threads_size);
+
+  for (int t = 0; t < threads_size; ++t)
```

Comment:
How about use `uint32_t` instead of `int` for safety? 
```suggestion
  for (uint_32_t t = 0; t < threads_size; ++t)
```
This can be applied to below for loops as well.